src/tutorials/0008-merkle-dags/01.md

```diff
@@ -0,0 +1,25 @@
+---
+    title: "Data has Structure!"
+    type: "multiple-choice" # can be "text", "code", "file-upload" or "multiple-choice"
+---
+
+In our Decentralized Data Structures tutorial, we discussed the
```

Comment:
May wish to change the name of the other tutorial now, given that it only talks about CIDs.

Reply:
I've renamed that one and updated references to it throughout the codebase.